Core/include/Acts/Seeding/PathSeeder.hpp

```diff
@@ -102,27 +115,27 @@ class PathSeeder {
   /// @arg The IP momentum magnitude
   /// @arg The particle charge
   using IntersectionLookup =
-      Delegate<std::vector<std::pair<GeometryIdentifier, Vector3>>(
+      Delegate<const std::vector<std::pair<GeometryIdentifier, Vector3>>(
```

Comment:
Return value const? Does that do anything in this case?

Reply:
Not in terms of safety. Just to keep things clean in the following iterations

Reply:
Not really. Probably an artifact of testing. Removed